Model/Quote/Checkout.php

```diff
@@ -97,6 +97,20 @@ public function getCheckoutSettings()
         ];
     }
 
+    /**
+     * @return float|int
+     */
+    private function getDeliveryDelayDay()
+    {
+        $productDelay = $this->package->dropOffDelayDayWithProduct($this->products);
+        $generalDelay = $this->helper->getIntergerConfig('general/dropoff_delay');
+
+        if (! $productDelay) {
+            $productDelay = $generalDelay;
+        }
+        return $productDelay;
```

Comment:
Het is meestal mooier als je een enter voor een return plaatst.

Reply:
done